app/views/finders/show_all_content_finder.html.erb

```diff
@@ -43,9 +44,24 @@
             } %>
           </div>
 
-          <div class="spelling-suggestions">
-            <%= render 'spelling_suggestion' %>
-          </div>
+          <% if @spelling_suggestion_presenter.suggestions.any? %>
+            <% suggestion = @spelling_suggestion_presenter.suggestions.first %>
+
+            <div class="app-all-content-finder__spelling-suggestions">
+              Did you mean <%= link_to sanitize(suggestion[:highlighted]), suggestion[:link],
```

Comment:
Is the source of highlighted quite trusted? Just wanted to check given sanitize is quite broad in what it accepts and I'm wondering if you only care about allowing a single element (mark) in and no attributes e.g. `sanitize(suggestion[:highlighted], tags: %w[mark], attributes: [])`

Reply:
Ah yes, this is just inlined from the old partial but no reason we can't improve things here 👍